webpage/app/pathPosIs.tsx

```
@@ -0,0 +1,24 @@
+import React from 'react';
+
+interface PathPosIsProps {
+    path: number;
+    position: number;
+    vertex: string;
+    onClick?: (path: number, position: number, vertex: string) => void;
+}
+
+const PathPosIs: React.FC<PathPosIsProps> = ({ path, position, vertex, onClick }) => {
+    return (
+        <div onClick={() => onClick?.(path, position, vertex)} className={`justify-around inline-flex flex-wrap-nowrap text-center border-2 rounded p-1 ${onClick !== undefined ? "cursor-pointer" : ""}`}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)